airbyte-webapp/src/pages/connections/ConnectionTransformationPage/DbtCloudTransformationsCard/DbtJobsForm.tsx

```diff
@@ -62,9 +62,14 @@ export const DbtJobsForm: React.FC<DbtJobsFormProps> = ({
                       <span className={styles.cardTitle}>
                         <FormattedMessage id="connection.dbtCloudJobs.cardTitle" />
                         <DropdownMenu
-                          options={availableDbtCloudJobs
-                            .filter((remoteJob) => !values.jobs.some((savedJob) => isSameJob(remoteJob, savedJob)))
-                            .map((job) => ({ displayName: job.jobName, value: job }))}
+                          options={
```

Comment:
Some thoughts here:
1) we have to put the logic which detects an empty job list upstream of the `.filter((remoteJob) => !values.jobs.some((savedJob) => isSameJob(remoteJob, savedJob)))`, so we don't show a "we can't find any jobs!" message to users when we've found the jobs just fine but filtered them all out
1) with the extra logic, I think we've hit the point where we need to pull the processed dropdown options to their own variable instead of doing it all inline.

Reply:
Mostly agree!

I think we can just conditionally render the dropdown/show some messaging if no jobs were found.